user-guide/notebooks/datasets/nldas_time_series.ipynb

```diff
@@ -94,602 +94,2438 @@
    "source": [
```

Comment:
<p>Not a blocker, but making a Bounding Box is simpler that hand coding a polygon. But this is also nice as a reference when it does need to be done.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>